tools/data-handler/src/project-settings.ts

```diff
@@ -101,7 +91,6 @@ export class ProjectSettings implements projectSettings {
    */
   public async commit() {
```

Comment:
Remove this function. Rename `persistConfiguration()`to `save()` and make it public. Replace all calls to `commit()` with `save()`

Reply:
Done in [f6a7188](https://github.com/CyberismoCom/cyberismo/pull/127/commits/f6a7188e2af5622d5ebefb247708de365aeaeef6)